Traducir.Core/Services/SOStringService.cs

```diff
@@ -596,5 +599,47 @@ Update Strings
             }
             return _strings;
         }
+
+        public async Task<bool> DeleteSuggestionAsync(int suggestionId, int userId)
+        {
+            using (var db = _dbService.GetConnection())
+            {
+                var idString = (int) await db.ExecuteScalarAsync(@"
+DECLARE @idString INT = 0;
```

Comment:
don't shout at the poor SQL Server, it's been good with us :)

Reply:
lol